hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/BackpressureRegulator.java

```diff
@@ -6,8 +6,9 @@
 import com.hazelcast.spi.Operation;
 
 import java.util.Random;
-import java.util.concurrent.atomic.AtomicInteger;
 
+import static com.hazelcast.client.impl.protocol.util.BitUtil.CACHE_LINE_LENGTH;
```

Comment:
Importing something from `com.hazelcast.client.impl.protocol....` seems not right to me. Even I find some methods and constants in `com.hazelcast.client.impl.protocol.util.BitUtil` redundant, we already have `com.hazelcast.nio.Bits`.